lib/ui/shared/preferences/preferenceswidgets/notification/preferences_notification_view.dart

```diff
@@ -245,7 +413,36 @@ class NotificationView extends StatelessWidget {
       }),
     ]);
   }
-}
 
 
+  
 
+  headerSpacer() {
+    return SizedBox(
+      height: 3,
+    );
+  }
+
+  buildHeaderAndSubTitle(String header, String subheader) {
```

Comment:
This is not valid too, no functional widget

Reply:
done